tests/components/elmax/test_config_flow.py

```diff
@@ -0,0 +1,268 @@
+"""Tests for the Abode config flow."""
```

Comment:
These are all partial tests. Please add a tests that cover the full flow. Additionally, if an user is tested during a flow, also add a user being able to recover from such error situation